exercises/acronym/description.md

```diff
@@ -4,5 +4,11 @@ Convert a phrase to its acronym.
 
 Techies love their TLA (Three Letter Acronyms)!
 
-Help generate some jargon by writing a program that converts a long name
-like Portable Network Graphics to its acronym (PNG).
+For example:
+
+|Input|Output|
+|-|-|
+|As Soon As Possible|ASAP|
+|Liquid-crystal display|LCD|
```

Comment:
I think I'd move these below the "Help generate" line.

Reply:
Moved to the end. Done.